lms/djangoapps/courseware/courses.py

```diff
@@ -453,32 +454,47 @@ def get_course_syllabus_section(course, section_key):
     raise KeyError("Invalid about key " + str(section_key))
 
 
-def get_courses(user, org=None, filter_=None):
+def get_courses(user, org=None, filter_=None, batch_size=100):
     """
     Return a LazySequence of courses available, optionally filtered by org code (case-insensitive).
     """
-    courses = branding.get_visible_courses(
+    courses_qset = branding.get_visible_courses(
         org=org,
         filter_=filter_,
+    ).select_related(
+        'image_set'
```

Comment:
Question: Is the ordering switch of brining `select_related` up in the chain significant here? It's fine either way, I'm just curious because I don't really understand the ORM internals well enough.

Reply:
No, it's not. Just happenstance of the path that brought me here, I'll revert so it's not confusing.